ansible/roles/test/files/ptftests/py3/hash_test.py

```diff
@@ -515,7 +515,7 @@ def check_within_expected_range(self, actual, expected, hash_key):
         '''
         percentage = (actual - expected) / float(expected)
         balancing_range = self.balancing_range
-        if hash_key == 'ip-proto' and self.topo_name == 't2':
+        if hash_key == 'ip-proto' and 't2' in self.topo_name:
```

Comment:
let's ignore this, it's already herer https://github.com/sonic-net/sonic-mgmt/pull/18459